playground/AspireEventHub/EventHubsConsumer/Consumer.cs

```diff
@@ -0,0 +1,33 @@
+using Azure.Messaging.EventHubs;
+using Azure.Messaging.EventHubs.Consumer;
+
+namespace EventHubsConsumer;
+
+/// <summary>
+///   Demonstrate consuming events from an Event Hub using the <see cref="EventHubConsumerClient" />.
+/// </summary>
+/// <param name="client"></param>
```

Comment:
Do we want to fill these out?

Reply:
The code under "playground" is just manual test apps. XML doc comments in that folder aren't required/used.